documentation/concepts/dryruns.md

```diff
@@ -28,8 +28,27 @@ all changes will be applied. Alternatively a separate tool of your choice can al
 directly to the database without using Flyway. This SQL file also contains the necessary statements to create and update Flyway's
 [schema history table](/documentation/concepts/migrations#schema-history-table), ensuring that all schema changes are tracked the usual way.
 
-This works transparently with all other Flyway features including SQL and Java migrations, both versioned and repeatable,
-callbacks, undo migrations, etc.  
+It is not advised to change a dry run script after it's been generated. Instead, any changes should be made to the migrations and a new dry run script generated. This is to ensure the changes executed match what's in your migrations.
+
+### Exceptions
+
+Not every change is intercepted in a Dry Run. Some changes cannot be intercepted and will be executed as normal. Details are provided;
```

Comment:
Is the semi-colon intended or should it be a colon? I don't remember much about punctuation 😬

Reply:
After some googling I think a colon is better 😛